lamp_virtual_host

```diff
@@ -64,16 +69,33 @@ check_required_package() {
   fi
 }
 #############################################################
+# Check OS
+#############################################################
+if [ -f /etc/redhat-release ]; then
+  printf "RUN Update\n"
+  sudo yum update
+  
+fi
+
+if [ -f /etc/lsb-release ]; then
+	printf "RUN Update\n"
+	sudo apt-get update && sudo apt-get upgrade
```

Comment:
basta solo update, in quanto upgrade se non metti -y chiede all'utente la conferma per procedere.